### ISSUE TITLE
Medallia accessibility improvements

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#43214]
We've identified the following a11y improvements for the Medallia UX on VA.gov. These will also align with the design and content standards on VA.gov.

On all screens, we recommend updating the font color to `#323A45`, in order to meet accessibility standards and improve contrast.

### Current state
![image](https://user-images.githubusercontent.com/55992116/175133612-95b58efc-8ba7-47f0-9143-14c0ecc2410e.png)

### Recommended improvements
![image](https://user-images.githubusercontent.com/55992116/175133726-d1c6a580-d071-4ce0-88f6-723b802f15fa.png)

### Current state
![image](https://user-images.githubusercontent.com/55992116/175134004-dc7a6200-7132-4e99-acd8-57cc2c12c456.png)

### Recommended improvements
- update the `font-weight` declaration on the confirmation page from 300 to 400 
- left align logo, thank you message, and close button (instead of center align)
- update the icon to improve contrast (further recommendation pending response on [this Slack thread](https://dsva.slack.com/archives/C01DBGX4P45/p1655930737122229))
- Remove the logo to preserve vertical height, bandwidth, and because it's decorative

Mock-up with logo (assuming we can't remove it for some reason)
![image](https://user-images.githubusercontent.com/55992116/175133365-8285aaea-7519-4df1-a048-1b36246f100d.png)

**Comment by jwoodman5** [comment]
These requests have been submitted to the Medallia Tech team for updates.

**Comment by andaleliz** [comment]
Hi @jwoodman5 checking in to see if there are any updates available.

**Comment by jwoodman5** [comment]
Hi @andaleliz yes, we were able to meet with the Medallia folks and segment these requests by ownership team. We have a few clarifying questions. Can you please help us by answering them? We plan to begin addressing this list in the upcoming sprint and following sprints.

1. **Request:** All font colors to #323A45.  **Question:** Does this include the button labels on the page? 
2. **Request:** Fonts for mobile are small, recommendation is 16 px. **Question:** Currently we have mixture of 14px for heading and 13px for answer options; Should all fonts be 16px?
3. **Request:** Open text PII disclaimer should move before the Input box. 
**Question:** Can you please confirm if you want the disclaimer before the text box (as shown in the screenshot above) or within the text box as placeholder (not sure if you knew this was an option)?

Thanks.

**Comment by andaleliz** [comment]
Hi @jwoodman5 sorry for the delay, this got buried in my inbox!

1. No, the button labels should remain aligned with the design system buttons
2. Yes, all fonts should be 16px
3. Confirming, we do want the PII disclaimer before the text box, not as a placeholder. Placeholders disappear as people type and that can cause confusion, particularly for folks with cognitive issues.  

Thanks for clarifying these points!

**Comment by jwoodman5** [comment]
Hi @andaleliz, now it's my turn to apologize for the delay. I was on PTO last week and had a heavy inbox on my return. We have been, however, making great progress here. We have 5 of the 8 issues resolved and are working on the remaining ones now. I will summarize status shortly, but wanted to get this note to you now. Thanks for your collaboration!

**Comment by andaleliz** [comment]
No problem! That's awesome that progress is being made, and I really appreciate your work on this. Have a great weekend :)

**Comment by jwoodman5** [comment]
Thank you, @andaleliz. We were able to deploy all requested updates today except for the following 2 items:
1. Changing the question `What is your overall satisfaction with this site` to `How did you feel about you visit to VA.gov today?`
2. Update the answer rating scale for this question (#1) from numbers to radio buttons with descriptors as shown above. (reporting will remain the same behind the scene and relative to 1-5 rating scale.)

For #1, it does not look like the proposed new question has been PRA approved as stated, so this would require further evaluation and likely PRA approval.
For #2, we just needed to coordinate this with our Medallia tech resource, which we did today, and that is now in-progress. I will let you know when that is done also. Have a great weekend!

**Comment by aubreyarcangel** [comment]
@andaleliz We have completed all the recommendations in this ticket.  All the CSS updates are in [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/22187) and the qualitative descriptions option answers and "Submit Feedback" copy change on the submit button were completed by the Medallia team.

**Comment by aubreyarcangel** [comment]
![Screen Shot 2022-10-13 at 10.06.38 AM.png](https://images.zenhubusercontent.com/5a8c8e864b5806bc2bc7b6c7/070ff54b-42ac-4d44-8d36-ccd7283974d4)

**Comment by aubreyarcangel** [comment]
![Screen Shot 2022-10-13 at 10.06.38 AM.png](https://images.zenhubusercontent.com/5a8c8e864b5806bc2bc7b6c7/070ff54b-42ac-4d44-8d36-ccd7283974d4)

**Comment by andaleliz** [comment]
This is such a strong improvement! Thank you for making these improvements.
